content/number-theory/Diophantine.h

```diff
@@ -0,0 +1,19 @@
+/**
+ * Author: Pietro Palombini
+ * Date: 2024-10-11
+ * Source: notebook el vasito
+ * Description: Returns $(x_0, y_0, dx, dy)$ such that all integer solutions (x, y) to $ax + by = r$
```

Comment:
```suggestion
 * Description: Returns $(x_0, y_0, dx, dy)$ such that all integer solutions $(x, y)$ to $ax + by = r$
```
También es medio raro que en una parte el producto este con un puntito y en otra no

Reply:
Es solo porque ahí todas las variables son de una letra. dx y dy son de dos.